addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderServerBridgeHandler.java

```diff
@@ -661,7 +668,10 @@ public void updateAvaliabilityStatus(IZoneMinderConnectionInfo connection) {
                     statusDetail = ThingStatusDetail.COMMUNICATION_ERROR;
                     statusDescription = "Session lost connection to ZoneMinder Server";
                     updateBridgeStatus(newStatus, statusDetail, statusDescription);
-                    logger.error("{}: {}", getLogIdentifier(), statusDescription);
+
+                    if (curBridgeStatus != ThingStatus.OFFLINE) {
+                        logger.error("{}: {}", getLogIdentifier(), statusDescription);
```

Comment:
Would it be acceptable to do some debug logging instead? Or is all types of logging no-go in this context?